openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -491,24 +491,88 @@ def _default_settings(cls):
         """
         return RelativeHybridTopologyProtocolSettings(
             protocol_repeats=3,
-            forcefield_settings=settings.OpenMMSystemGeneratorFFSettings(),
+            forcefield_settings=settings.OpenMMSystemGeneratorFFSettings(
+                # fast settings
+                nonbonded_cutoff=0.9 * unit.nanometer
+            ),
             thermo_settings=settings.ThermoSettings(
                 temperature=298.15 * unit.kelvin,
                 pressure=1 * unit.bar,
             ),
             partial_charge_settings=OpenFFPartialChargeSettings(),
-            solvation_settings=OpenMMSolvationSettings(),
+            solvation_settings=OpenMMSolvationSettings(
+                # fast settings
+                box_shape="dodecahedron",
```

Comment:
Same thing here, can you update this in the original settings object? that way we can update septop/abfes at the same time automatically